lib/AST/UnqualifiedLookup.cpp

```diff
@@ -485,12 +485,7 @@ void UnqualifiedLookupFactory::performUnqualifiedLookup() {
       DC, initialIsCascadingUse};
   const bool crosscheckUnqualifiedLookup =
       Ctx.LangOpts.CrosscheckUnqualifiedLookup;
-  // SWIFT_ENABLE_TENSORFLOW
-  // NOTE(TF-815): using AST scopes for lookup causes standard library
-  // type-checking for `@differentiable` attributes to fail.
-  if ((false)) {
-  // if (useASTScopesForLookup()) {
-  // SWIFT_ENABLE_TENSORFLOW END
+  if (useASTScopesForLookup()) {
```

Comment:
Note: this reverts an [earlier workaround hack](https://github.com/apple/swift/commit/69ab762d68f8fc6564eda3f4390849e492b06e48).